### ISSUE TITLE
General Size Definitions

### ISSUE THREAD

**Issue by Gedan** [Fenoxo/sourceTiTS#115]
This has actually been something that I keep running into in CoC, and now that I'm working on actual content in TiTs it's reminding me to bring it up.

Rather than leaving effectively random floating values for various size descriptions in places, can we come up with some kind of generically defined "cheat table" for size comparisons?

By that I mean, how we constitute say, a cock large enough to bulge an NPCs gut, or what is considered a "large" cumshot vs an "extreme" cumshot.

Leaving things as just raw values that we have defined as globals is *probably* not going far enough. A cock big enough to bulge a 5' tall characters belly would probably be much smaller than a cock required to bulge a 12' tall characters belly etc.

Basically, I'm thinking we need to aim towards some kind of value-comparison and build the thing as a set of boolean methods built into creature classes. From there things would operate similar to how loadIn<Hole> works; something like ```kiro.isBellyBulged(pc.cockVolume(desiredCock))``` or ```pc.isLargeCumQ(kiro)``` -- that would let the calls adapt to the general differences between involved characters. 

Yes, I have a boner for shit being consistent. I think in this case it could only be a benefit, as it'd move us away from "Huehue pull numbers outta my butt" whenever shit comes up.
